core/src/data/value/selector.rs

```diff
@@ -45,4 +45,13 @@ impl Value {
 
         Ok(value)
     }
+
+    pub fn selector_by_index(&self, selector: Vec<Value>) -> Result<Value> {
+        let keys = selector
+            .iter()
+            .map(String::from)
+            .collect::<Vec<_>>()
+            .join(".");
+        self.selector(&keys)
+    }
```

Comment:
I just converted `Vec<Value>` into form of **"index1.index2.index3"** instead of modifying `selector` fn (which used for `unwrap` fn). Since this is just a tmp solution, I need suggestions for it.